cms/server/admin/handlers/user.py

```
@@ -29,35 +29,177 @@
+        old_password = user.password if user else None
+        self.get_password(attrs, old_password, False, allow_weak)
+        self.get_string_list(attrs, "preferred_languages")
+        self.get_string(attrs, "timezone", empty=None)
+
+        # Handle date of birth
+        date_of_birth_str = self.get_argument("date_of_birth", "").strip()
+        if date_of_birth_str:
+            try:
+                attrs["date_of_birth"] = date.fromisoformat(date_of_birth_str)
+            except ValueError:
+                raise ValueError("Invalid date of birth format") from None
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)